klog.go

```diff
@@ -432,6 +433,7 @@ func InitFlags(flagset *flag.FlagSet) {
 	flagset.BoolVar(&logging.toStderr, "logtostderr", logging.toStderr, "log to standard error instead of files")
 	flagset.BoolVar(&logging.alsoToStderr, "alsologtostderr", logging.alsoToStderr, "log to standard error as well as files")
 	flagset.Var(&logging.verbosity, "v", "number for the log level verbosity")
+	flagset.BoolVar(&logging.skipHeaders, "add_dir_header", logging.addDirHeader, "If true, adds the file directory to the header")
```

Comment:
Shouldn't this be `&logging.addDirHeader`?

Reply:
yes, it should. cc @dims

Reply:
@jhahn21 Could you please file a PR to fix it. good catch. thanks!

Reply:
@dims Sure, I will.

Reply:
@dims Here is the PR: https://github.com/kubernetes/klog/pull/101